src/Core/Content/Seo/SeoUrlPlaceholderHandler.php

```diff
@@ -111,6 +127,50 @@ private function createSeoMapping(SalesChannelContext $context, array $mapping):
         return $mapping;
     }
 
+    private function createMediaMapping(SalesChannelContext $context, array $mapping): array
+    {
+        $prefix = '/mediaId/';
+
+        if (empty($mapping)) {
+            return [];
+        }
+
+        $mediaIds = [];
+        foreach ($mapping as $item) {
+            if (str_starts_with($item, $prefix)) {
+                $mediaIds[] = substr($item, \strlen($prefix));
+            }
+        }
+
+        $urls = $this->getMediaUrls($mediaIds, $context->getContext());
+
+        foreach ($urls as $id => $url) {
+            $key = self::DOMAIN_PLACEHOLDER . $prefix . $id . '#';
+            $mapping[$key] = $url;
+        }
+
+        return $mapping;
+    }
+
+    private function getMediaUrls(array $mediaIds, Context $context): array
```

Comment:
yeah, good point. I think we can depend on #2692 if that has change to be merged soon

Reply:
#2692 is taking more time than expected, still this would be a very useful feature. 

@shyim Can you elaborate on what meant with the first option:

> The admin links by the given URL

Reply:
@shyim ping :-) Wondering if I should wait for #2692 or you had an alternative solution in mind :-)

Reply:
@shyim 

> The admin links by the given URL

which this you mean that we persist the fully resolved URL in the content? Reminds me a bit of WordPress ;-)
At least we need to store the original UUID to be able to show the selector popup?


Any other ideas? Do you have eventually information about the status of https://github.com/shopware/platform/pull/2692